integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/package.scala

```diff
@@ -87,8 +87,8 @@ abstract class AtomicRunnableCommand
   extends RunnableCommand with MetadataProcessOpeation with DataProcessOperation {
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
-    processMetadata(sparkSession)
```

Comment:
This change may impact other commands. Better don't include in try block. Let the individual command handle it.